cypress/component/annotation-explanation.cy.js

```diff
@@ -0,0 +1,32 @@
+import annotExplanation from "~/components/annot-explanation";
+
+describe('explanation', () => {
```

Comment:
Just a small change to maintain consistent quotation characters throughout. We've gone with double quotation marks elsewhere.

Reply:
Thanks, good catch. Not sure why this wasn't flagged by the precommit linter.